crypto-ffi/bindings/jvm/src/test/kotlin/com/wire/crypto/MLSTest.kt

```diff
@@ -572,4 +572,158 @@ class MLSTest : HasMockDeliveryService() {
             assertNotEquals(results1[0], results2[0])
         }
     }
+
+    @Test
+    fun can_create_keypackage(): TestResult {
+        val scope = TestScope()
+        return scope.runTest {
+            val clientId = genClientId()
+            val credential = Credential.basic(CIPHERSUITE_DEFAULT, clientId)
+
+            val cc = initCc(this@MLSTest)
```

Comment:
What does `@MLSTest` do?

Reply:
It's a Java idiom meaning "pass `this`, but treat it as a `MLSTest` instance instead of whatever super/subclass you might otherwise have used by default". I don't remember why specifically, but we've needed that kind of thing for `initCc` for a while now.